### ISSUE TITLE
Table i and j headings

### ISSUE THREAD

**Issue by andrew-edwards** [cgrandin/hake-assessment#248]
Column heading
 
'Probability Fishing intensity in 2017 >40% Target' 

should presumably be

'Probability relative fishing intensity in 2017 >100%'

and similarly for Table j (and duplicated Tables 23 and 24).

And what should final column be? Replace 'Target' with 'option' (or limit, which is what caption says)?

I'm off shortly so won't change anything.

**Comment by andrew-edwards** [comment]
First part also could do with fixing in Figures j and k caption for the red lines (and later figures if they're duplicated)

**Comment by aaronmberger** [comment]
Hmm, I think we should leave it in reference to the 40% target here, but change the language a bit as you suggest.  Changing to the following in applicable tables:
Column heading: Probability 2017 fishing intensity > FSPR40%
Column heading: Probability 2018 Catch at FSPR40% < 2017 Catch
Figures j and k and 40 and 41 legends adjusted accordingly also 

I'll work on this now if good

**Comment by andrew-edwards** [comment]
Agree at using FSPR40% (and not saying target). FSPR40% (\Fforty I think) is defined earlier, so that's good.

Second heading - though isn't is actually the Prob 2018 catch option < 2017 catch option as defined for each row?
Hmm... no, that's wrong since the constant catch options have no probabilities associated with them. So, yes, must be what you say. Is the fact that h gives 50% in Table i just a coincidence (as option h's definition kind of implies a 50% results for my incorrect interpretation above). [Think I'm misunderstanding option h right now, will think later...]
So, yes, I think you're good!

**Comment by andrew-edwards** [comment]
Hmm... that value for row h was 50% last year also, so it's not a coincidence.
I think I know what I"m not understanding - just looking at your forecast-find-decision-tables.r code which should give me the answer....

**Comment by andrew-edwards** [comment]
Hmm... if you say catch in 2017 and 2018 is fixed at 866,263 then where does the probability come in?
I seem to remember someone saying that it's done by varying F, not the actual amount caught, so you can end up with a probabilistic statement.
So is option h better defined (in Table g) as:
"the fishing intensity* that results in a 50% probability ... "
Aha - I think that's it?!?!

* I originally thought 'fishing mortality rate', but maybe it is done as fishing intensity. Only occurrence of 'fishing mortality rate' in exec-summary is mine that I did earlier when defining F_{SPR=40%} etc.

**Comment by aaronmberger** [comment]
Owen provided a comment on this and highlights the "2018 Catch Target" part and says "here this means default target rather than actual target" so I'm taking from that the 2018 catch target is referring to the default harvest policy.  so i'm currently thinking (and this has been evolving) that this is measuring the probability that the 2018 catch as defined by the default harvest policy will be less than in 2017 under options a-h.  It kind of makes sense that option g refers the the default policy for 2017 so if take that much catch there should be a > 50% change that in the next year the stock will go down given we are well below a FSPR=40% level.

**Comment by aaronmberger** [comment]
just committing a change, see what you think...

**Comment by andrew-edwards** [comment]
Think commit is good. 
Though should penultimate column's 'fishing intensity' be 'fishing mortality rate'?  I don't think you can compare a fishing intensity with a fishing mortality rate. And we don't have an abbreviation for fishing intensity. 
And I wonder if my suggestion above about defining option h should indeed be 'fishing mortality rate'? Bleh, going round in circles. I'll leave this for now and get back in the morning. Think I'm doing 'negative work' (as in work that will need to be corrected in the morning!). Almost there with these definitions though....

**Comment by taylori** [comment]
Good job Andy and Aaron with all the progress you made last night.

Here are some thoughts on the Table i and j headers, the last two of which are truly confusing concepts.

First, I looked at the code in the calc.risk function in load-models.r.
It is very simple. The last two columns are calculated as
    out[6] <- sum(x[, paste0("SPRratio_", yr)] > 1.00) / nrow(x) * 100.0
    out[7] <- sum(x[, paste0("ForeCatch_", yr + 1)] < out[1]) / nrow(x) * 100.0
where out[1] is the catch for the year before.

So here's what I think they are saying:
Column 6 (penultimate column) in Table i: If you fix the 2017 catch at this value, and then calculate the distribution of relative fishing intensity that results for 2017,  what fraction is above 100%. Row f should always have a probability of 50% as Andy notes.
Column 6 in Table j. If you fix the 2017 catch at the value shown in Table i and then fix the 2018 catch at the value shown in Table j, what fraction of the distribution of relative fishing intensity is above 100%.
Since there's a 1-to-1 mapping from SPR to fishing intensity and relative fishing intensity, I think P(SPR<0.4)  = P(1-SPR>0.6) = P(Rel. Fish. Int > 100%). The old caption was definitely wrong under any either old or new definition of fishing intensity. My vote for both tables would be "Probability Relative Fishing Intensity in YYYY > 100%".

Column 7 (final column) in Table i: If you fish the 2017 catch at this value, and project the model forward with uncertainty, and apply the default harvest control rule to calculate a catch limit for 2018, what's the change that the 2018 default control rule limit will be less than the number you fixed for 2017. By definition, row h should always be 50% in this final column. I think it's wrong to say "Catch Target", so I would change to something like "Probability 2018 Catch from Default Control Rule < 2017 Catch"

Column 7 in Table j: Same idea. If you set the 2017 and 2018 catches at the values shown in Tables i and j, and then you calculate the 2019 default control rule catch limit, what's the chance that the value for 2019 will be lower than the fixed 2018 value. I guess I would make the same suggestion above. What's extra confusing here is row h. The catch stream in row h is based on finding the fixed catch in 2017 that gives you the same value as the median default catch in 2018, which has no regard for what happens in 2019. By 2019 the stock is likely to be declining due to the big cohorts dying off and catching almost 1 million tons per year in this catch stream, so it makes sense that the 2019 default control rule is likely to be lower than the catch in 2018 (which matched the catch in 2017).

Hopefully this helps.
In summary, my suggestion is to change the headings in Tables i and j to
Col 6: "Probability Relative Fishing Intensity in YYYY > 100%"
Col 7: "Probability YYYY+1 Catch from Default Control Rule < YYYY Catch"

There's also room to expand the captions for Table i and j if we want to clarify what the columns mean.

**Comment by aaronmberger** [comment]
Ok, I'll make those changes now...  nice detective work Ian!

**Comment by andrew-edwards** [comment]
Excellent - we got there! Thanks Ian for looking into the code.

One minor thing more (as noted above), so is option h better defined (in Table g) as:
"the fishing intensity that results in a 50% probability ... "
rather than the current
"the catch level that results in a 50% probability ..."
Yes?

**Comment by aaronmberger** [comment]
See recent comitt - will keep open until someone double checks it

**Comment by aaronmberger** [comment]
closing, but can be re-opened if someone doesn't like changes but they should reflect our collective agreement (at least for now.....)

**Comment by andrew-edwards** [comment]
Yes, I think it's all good (just forgot to comment)
